mms/model_service_worker.py

```diff
@@ -154,5 +154,8 @@ def run_server(self):
         logging.error("Backend worker did not receive connection in: %d", SOCKET_ACCEPT_TIMEOUT)
     except Exception:  # pylint: disable=broad-except
         logging.error("Backend worker process die.", exc_info=True)
+    finally:
+        if sock_type == 'unix' and os.path.exists(socket_name):
+            os.remove(socket_name)
```

Comment:
This is not consistent with earlier one: os.unlink()
Can we change both to os.remove()?